NFIQ2/NFIQ2Algorithm/include/nfiq2/tool/nfiq2_ui_refresh.h

```diff
@@ -24,6 +27,20 @@
 
 namespace NFIQ2UI {
 
+bool quantizeCheck();
+
+bool defaultCheck(const std::string &name, const uint16_t defaultDPI,
```

Comment:
```suggestion
bool askToResampleIfNoResolution(const std::string &name, const uint16_t defaultDPI,
```

Hmm, maybe you can think of a better name.